### PR TITLE
Bring back plans features row highlighting

### PR DESCRIPTION
**Pull request by lamosty**
Fix for #7160. Fixes the plan features highlighting.

## Testing Instructions

1. Navigate to `/plans/:site` and make sure everything works as before;
2. Navigate to `http://calypso.localhost:3000/plans/lamostypremium.wordpress.com?feature=no-wp-branding` and make sure you see the "Remove WordPress.com Branding" feature highlighted.

### With all three columns highlighted

selection_019

### With one column highlighted

selection_020

## Todo

- [ ] Add highlighting for Jetpack plans

cc @gwwar @artpi 

Test live: https://calypso.live/?branch=fix/bring-back-plans-features-row-highlighting

### CODE REVIEW

File: client/my-sites/plan-features/style.scss [hunk 1]
```diff
@@ -91,6 +91,26 @@ $plan-features-sidebar-width: 272px;
 	}
 }
 
+.plan-features__table-row {
+	&.is-highlighted {
+		background-color: rgba($alert-yellow, 0.1);
+		td {
```
Comment (mtias) [inline client/my-sites/plan-features/style.scss]: it'd be good if all these had classes to target.

File: client/my-sites/plan-features/style.scss [hunk 2]
```diff
@@ -91,6 +91,26 @@ $plan-features-sidebar-width: 272px;
 	}
 }
 
+.plan-features__table-row {
+	&.is-highlighted {
```
Comment (mtias) [inline client/my-sites/plan-features/style.scss]: In this case, should just write it as `.plan-features__table-row.is-highlighted {}`

File: client/my-sites/plan-features/style.scss (hunk 1 above)
Comment (lamosty) [inline client/my-sites/plan-features/style.scss]: They have: `plan-features__table-item`.

Comment (mtias) [inline client/my-sites/plan-features/style.scss]: Let's use that then.

Comment (lamosty) [inline client/my-sites/plan-features/style.scss]: Note to self: don't blindly copy other's code...

Comment (mtias) [inline client/my-sites/plan-features/style.scss]: There's always an opportunity to improve things :)

File: client/my-sites/plan-features/style.scss [hunk 3]
```diff
@@ -96,10 +96,14 @@ $plan-features-sidebar-width: 272px;
 	border-left: solid 1px lighten( $gray, 27% );
 	background-color: lighten( $gray, 35% );
 	position: relative;
-}
 
-.plan-features__table-item.is-selected {
-	display: table-cell;
+	&.is-highlighted {
+		border: 1px solid $alert-yellow;
```
Comment (mtias) [inline client/my-sites/plan-features/style.scss]: Should also bring the `background-color: rgba( $alert-yellow, 0.1 );`

**Comment by mtias** [comment]
@lamosty I think this looks good to go.

**Comment by lamosty** [comment]
@mtias thanks for the review and help! I still need to fix feature highlighting for Jetpack plans. Will do it today in the evening (hopefully) and then we can merge.

**Comment by mtias** [comment]
I don't think feature highlighting for Jetpack is needed now since they don't use upgrade-nudges yet.

**Comment by lamosty** [comment]
> I don't think feature highlighting for Jetpack is needed now since they don't use upgrade-nudges yet.

It was already there but I forgot to pass the `selectedFeature` prop all the way down to Jetpack plans renderer. :D

**Comment by gwwar** [comment]
:shipit:

**Pull request merged**